stylesheets/_layout.settings.scss

```diff
@@ -0,0 +1,439 @@
+    margin-left: 0;
+    width: 100%;
+
+    &::before {
+        @include respond-min($screen-smaller) {
+            content: '';
+            display: table-cell;
+            width: 40%;
+        }
+    }
+}
+
+.tab__settings .form__group--indent .controls {
+    display: table-cell;
+}
+
+
+/**
+ * Form group labels padding adjustments for smaller views
+ */
+
+.tab__settings .form__group:not(.form__group--top):not(.form__group--toggle) > .control__label {
+    @include respond-max($screen-smaller) {
+        padding: 0 0 $padding-base-vertical;
+    }
+
+    @include respond-min-max($screen-smaller, $screen-medium) {
+        padding-right: $padding-small-horizontal;
+        width: 35%;
+    }
+}
+
+
+/**
+ * Form control file uploads alignment improvements
+ */
+
+.tab__settings .form__control.file {
+    height: $line-height-large;
+    padding-top: 14px;
+    width: 100%;
+
+    @include respond-min($screen-small) {
+        max-width: 100%;
+        width: $controls-max-width;
+    }
+}
+
+
+/**
+ * Guidance popover icon background colour should match settings column background colour
+ */
+
+.tab__settings .input-group-guidance {
+    background-color: color(gray, off-white);
+}
+
+
+/**
+ * Content area
+ */
+
+.has-settings .tab__content {
+    @include respond-min($screen-small) {
+        margin-bottom: $line-height-large;
+        padding-left: $padding-small-horizontal * 2;
+    }
+
+    @include respond-min($screen-medium) {
+        float: right;
+        max-width: calc(99% - 370px);
+        width: 74%;
+    }
+
+    .lt-ie9 & {
+        width: 64%;
+    }
+}
+
+
+/**
+ * Content header
+ */
+
+.content-header {
+    @include clear-fix(true);
+
+    border-bottom: 1px solid color(gray, lighter);
+    padding-top: $padding-base-vertical * 2;
+
+    .is-fixed & {
+        background-color: color(white);
+        position: fixed;
+        top: 0;
+        z-index: 101;
+    }
+}
+
+
+/**
+ * Content heading
+ */
+
+.content-heading {
+    display: block;
+    line-height: $input-height-small;
+    margin-bottom: $padding-base-vertical * 2;
+}
+
+
+/**
+ * Content actions
+ */
+
+.content-actions {
+    float: right;
+    margin: 0 0 ($padding-base-vertical * 2) $padding-large-horizontal;
+}
+
+.content-actions__pages,
+.content-actions__buttons {
+    display: inline;
+}
+
+.content-actions__pages .form__group,
+.content-actions__pages .controls,
+.content-actions__pages .form__group .form__control:not(.checkbox):not(.radio):not(.file) {
+    display: inline;
+    width: auto;
+}
+
+.content-actions__pages .form__group {
+    margin: 0;
+    table-layout: auto;
+}
+
+.content-actions__pages .form__control {
+    height: $input-height-small;
+    line-height: $input-height-small;
+}
+
+.content-actions__buttons {
+    margin-left: $padding-small-horizontal;
+}
+
+.content-actions__buttons .btn {
+    vertical-align: top;
+}
+
+
+/**
+ * No results component should span full width of content area
```

Comment:
Would this be more appropriate with the other `.no-results` styles in `_utility.utilities.scss`?